iroha-balancer-core/src/test/java/soramitsu/irohautils/balancer/IrohaBalancerCoreContainerTest.kt

```diff
@@ -0,0 +1,56 @@
+package soramitsu.irohautils.balancer
+
+import iroha.protocol.TransactionOuterClass
+import jp.co.soramitsu.iroha.java.Transaction
+import jp.co.soramitsu.iroha.java.Utils
+import jp.co.soramitsu.iroha.testcontainers.detail.GenesisBlockBuilder
+import org.junit.Assert
+import org.junit.ClassRule
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.testcontainers.containers.Network
+import soramitsu.irohautils.balancer.helper.ContainerHelper
+import java.util.*
+import javax.xml.bind.DatatypeConverter
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class IrohaBalancerCoreContainerTest {
+
+    private val containerHelper = ContainerHelper()
+
+    private val irohaBalancerCoreContextFolder = "${containerHelper.userDir}/build/docker/"
+    private val irohaBalancerCoreDockerfile = "${containerHelper.userDir}/build/docker/Dockerfile"
```

Comment:
it seems there must be a dependency in the gradle file so that `dockerFileCreate` will always run before `test`

Reply:
Agree. Do you have any examples of that?

Reply:
https://docs.gradle.org/current/userguide/more_about_tasks.html Example 18
kinda `test.dependsOn dockerFileCreate`